modules/ROOT/pages/obtain-connected-apps-credentials.adoc

```diff
@@ -8,7 +8,12 @@ Connected Apps enables you to seamlessly integrate external applications with An
 
 To create a Connected App and obtain its credentials:
 
-. From Anypoint Platform, select *Access Management* > *Connected Apps* > *Create App*.
+. Log into Anypoint Platform and select *Access Management*.
+. Ensure that you are viewing the menu for your Master organzation in the left navigation pane.
```

Comment:
"in the left navigation pane" seems to be in the wrong place. How about "Look in the left navigation pane to ensure that you are viewing the menu..."?

Reply:
Done! Thanks!